src/main/webapp/resources/js/renderCalendar.js

```diff
@@ -188,9 +196,16 @@ $(function () {
     });
 
     $('#update-recurrent-button').click(function () {
-        if(validateEventDialogData(UPDATE_RECURRENT_EVENT)){
```

Comment:
checking that validation for each type of radio-button. inside validateEventDialogData for CREATE_RECURRENT_EVENT && UPDATE_RECURRENT_EVENT we have the same code. and actually updating is deleting existent event + creating new one